src/site/content/en/react/how-to-use-amp-in-nextjs-and-why/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: How to use AMP in Next.js (and why)
+authors:
+  - houssein
+subhead: |
+  Add AMP to any of your Next.js pages to guarantee fast page loads
+date: 2019-10-17
+draft: true
+---
+
+[AMP](https://amp.dev) (Accelerated Mobile Pages) is a web component framework that guarantees
+that your pages will load fast. [Next.js][intro] has built-in support for AMP.
+
+## What will you learn?
+
+This guide first briefly describes [how AMP guarantees fast page loads](#overview), then lets you
+try out [the two ways that you can use AMP in a Next.js app](#strategies), then helps you [decide
+which approach is best for you](#guidance).
+
+The intended audience for this guide is a web developer who has decided to use Next.js but is
+unsure of whether to support AMP. 
+
+{% Aside %}
+  This guide was not written for web developers who have decided to use AMP but are unsure of what
+  framework to use. We will note briefly however that Next.js could be a good choice because it
+  supports [AMP server-side rendering](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering/)
+  and makes it easy to serve AMP content without introducing a lot of complexity into your codebase.
+{% endAside %}
+
+## How AMP guarantees fast page loads {: #overview }
+
+AMP has two main strategies for guaranteeing fastness:
+
+* **AMP HTML**: A restricted form of HTML that makes certain optimizations mandatory and prohibits
+  architectural patterns that lead to slowness. See [How AMP works][how] for a high-level
+  overview of the optimizations and restrictions.
+* **AMP Cache**: A content cache used by some search engines, such as Google and Bing, that uses
+  [prerendering] to speed up page loads. See [Why AMP Caches exist][why] to learn more about the
```

Comment:
Link for prerendering missing?

Reply:
This is Markdown shorthand. When the text matches the link ID, you can just do `[prerendering]` instead of `[prerendering][prerendering]`. Similar to ES6 object shorthand syntax.